src/zope/viewlet/viewlet.py

```diff
@@ -149,12 +162,20 @@ class ResourceBundleViewletBase(object):
     """
     _paths = None
 
+    #: A callable (usually a template) that is used to implement
+    #: the `render` method.
+    index = None
+
     def getResources(self):
+        """
+        Lookup all the resources in our desired paths using the
```

Comment:
Not a native English speaker, but I think the verb form is "Look up".

Reply:
Indeed, the dictionary agrees with you. That usage must be colloquial or dialectal. Thanks!

Reply:
FWIW there were other occurrences of this, but my headache said 'not now' and I didn't mark them out.

Reply:
Oh, I see you changed all three that I remember noticing.  Thank you!

Reply:
I did a search and believe I corrected all the instances I introduced. There are still uses of 'lookup' as a verb in preexisting text in directives.rst, metadirectives.py and README.rst.